pkgs/tools/misc/colorless/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, fetchurl
+, stdenvNoCC
+, coreutils
+, dash
+, gnused
+, less
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "colorless";
+  version = "109";
+
+  src = fetchurl {
+    url = "http://software.kimmo.suominen.com/${pname}-${version}.tar.gz";
+    sha256 = "039a140fa11cf153cc4d03e4f753b7ff142cab88ff116b7600ccf9edee81927c";
+  };
+
+  makeFlags = [
+    "TOOLPATH=${coreutils}/bin:${gnused}/bin:${less}/bin"
+    "PREFIX=${placeholder "out"}"
+    "SHELL=${dash}/bin/dash"
+  ];
+
+  preInstall = ''
+    mkdir -p $out/bin $out/share/man/man1
+  '';
+
+  postInstall = ''
+    mkdir -p $out/share/doc/colorless
+    install -Dm644 -t $out/share/doc/colorless LICENSE README.md
+  '';
```

Comment:
Is there some reason this isn't part of the package's `make install` process?

Reply:
Traditionally I've had `make install` only install the functional parts of the software.

With binary packages it is convenient and useful to include additional content from the software distribution, but I've always felt that it is up to the packaging system to choose what is appropriate for each one of them.  However, you raise a good point, considering that packaging systems and binary packages are probably the most common distribution channels nowadays.  As a software author and maintainer, I should probably exercise more control over what is and/or should be included in binary packages.

I will take this under consideration for the next release of `colorless` as well as other software packages I maintain.